src/plots/power_vega.jl

```diff
@@ -143,19 +155,21 @@ end
 
 
 
-function plot_vega( case::Dict{String,<:Any};
+function plot_vega( case;
                     spring_constant::Float64=1e-3,
-                    color_symbol=:ComponentType,
-                    kwargs...
+                    bus_color=:green,
+                    gen_color=:blue,
+                    branch_color=:black,
+                    dcline_color=:steelblue,
+                    connector_color=:gray,
+                    color_symbol=:ComponentType
```

Comment:
These changes are removing the plot_attributes features from the dev branch.   You said you rebased it already?

You might want to just copy and paste the dev branch version of plot_vega into you're pull request to prevent these reversions.